osu.Game.Rulesets.Osu/Mods/OsuModSuddenDeath.cs

```diff
@@ -13,5 +18,16 @@ public class OsuModSuddenDeath : ModSuddenDeath
         {
             typeof(OsuModTargetPractice),
         }).ToArray();
+
+        [SettingSource("PFC mode", "Fail when missing on a slider tail")]
+        public BindableBool SliderTailMiss { get; } = new BindableBool();
+
+        protected override bool FailCondition(HealthProcessor healthProcessor, JudgementResult result)
+        {
+            if (SliderTailMiss.Value && result.HitObject is SliderTailCircle && result.Type == HitResult.IgnoreMiss)
+                return true;
+
+            return result.Type.AffectsCombo() && !result.IsHit;
```

Comment:
Should just return base.

Reply:
Does https://github.com/ppy/osu/pull/31232/commits/5c9278ee2f5c044e0b6565973497b559f620ab5e properly address this problem?

Reply:
What I meant was to change line 30 to `return base.FailCondition(healthProcessor, result)`.

Reply:
Ok, should be fixed in https://github.com/ppy/osu/pull/31232/commits/047c448741a6c2ab038a04085ebab97048e8473d